R/input_output_filter.R

```diff
@@ -229,6 +245,10 @@ filter_samples_by_counts = function(input, min_seqs) {
 #'  values.
 #' @return A list variable with (1) the loaded dissimilarity matrix and (2) 
 #'  the loaded mapping file.
+#' @examples 
+#' \dontrun{
+#' filtered_dm = filter_dm(loaded_dm, "Farm_type", filter_vals = "organic")
```

Comment:
Note that input variables should actually exist when available or if they can be easily created. So you could change 'loaded_dm' and before it include another line with a function that actually creates the dm. You would then be able to remove the \dontrun command because it would actually run.